RFC-000-rfc-process-suggestions.md

```diff
@@ -0,0 +1,77 @@
+# RFC Zero - Open for suggestions to the RFC process
+
+## Proposed by
+
+[Alyssa Rock](https://thegooddocs.slack.com/team/U012KCMPP0V)
+
+## Current status
+
+- [x] Draft
+- [x] Under discussion (until 2021-07-28)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+
+## Proposal overview
+
+This RFC is open to suggestions from the community for minor and major suggestions to improve the RFC process.
+To make your suggestion to the process, add a comment to the "Feedback" section of this file or to this PR.
+
+
+## Motivation
+
+We've completed the RFC process with at least one RFC now and we've learned a few lessons along the way.
+Some members of the community expressed a desire to make a few minor tweaks and improvements and this RFC is designed to facilitate that discussion.
+
+
+## Proposal
+
+Make your suggestions here.
+Near the end of the community discussion period, I'll take the suggestions and add them to this section.
+After that, we'll open the final comment period and discuss the final changes.
+
+
+## Links
+
+No links necessary!
+

```

Comment:
### Section Proposal Potentials

These are sections I've used in other RFC processes I found helpful

- Open Questions
- Decisions Deferred
- Consequences

Reply:
Good idea! I've added this to the RFC.